src/components/ToggleListItem.js

```diff
@@ -56,7 +56,7 @@ export const ToggleListItem = ({ item, index, section }) => {
       onPress={onPress}
       delayPressIn={0}
       Component={Touchable}
-      accessibilityLabel={`${title} (Taste)`}
+      accessibilityLabel={`(${title})${consts.a11yLabel.button}`}
```

Comment:
please double check having spaces for correct written language in these cases

Reply:
done in 71c498a949d6f531126e672bc262773d65a2cbf2